spec/menu_printer_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "spec_helper"
+require "menu_printer"
+
+RSpec.describe MenuPrinter do
+  describe "#edit_project_menu" do
+    it "given an existing project, prints out the project's name in the options" do
+      stdout = StringIO.new("")
+      project_name = "Build tiny home"
+      menu = MenuPrinter.new(stdout)
+      output = <<~MENU
+      Editing project: 'Build tiny home'
```

Comment:
Small nit about formatting: it would make this easier to read if the contents were indented. Since you are using the ~ version of heredocs, it should trim leaving white space for you!